### PR TITLE
Translator stopwatches 1.17.x

### PR DESCRIPTION
**Pull request by ashishb-solo**
# Description

Changelog entry to document [this pull request](https://github.com/solo-io/gloo/pull/10243) being merged into 1.17.x. That pull request lacked a changelog entry, so this pull request simply adds one for that work to support our version documentation.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/10278

### CODE REVIEW

**Review by nfuden — APPROVED** [review]

File: changelog/v1.17.16/translator-stopwatches-1.17.x.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/10278
+    resolvesIssue: true
+    description: >-
+      Add stopwatches for translator and Envoy syncer.
+
+      The work for this pull request was done in
+      https://github.com/solo-io/gloo/pull/10243 - that pull request lacked a
+      changelog. This pull request simply adds the changelog entry that was
+      missing from the other pull request.
```
[the remark this answers is not in the document]
Comment (nfuden) [reply changelog/v1.17.16/translator-stopwatches-1.17.x.yaml:11]: Thats not user facing

File: changelog/v1.17.16/translator-stopwatches-1.17.x.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/10278
+    resolvesIssue: true
+    description: >-
+      Add stopwatches for translator and Envoy syncer.
```
Comment (nfuden) [inline changelog/v1.17.16/translator-stopwatches-1.17.x.yaml:6]:
```suggestion
      Add metrics for translatation timing and time for taken to sync served xDS entries.
```

File: changelog/v1.17.16/translator-stopwatches-1.17.x.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashishb-solo) [reply changelog/v1.17.16/translator-stopwatches-1.17.x.yaml:11]: Are you suggesting that I take this out? At the very least, I would still like to keep the link to https://github.com/solo-io/gloo/pull/10243 to show the code that was merged for the fix...

[the remark this answers is not in the document]
Comment (nfuden) [reply changelog/v1.17.16/translator-stopwatches-1.17.x.yaml:11]: Can it be in the PR description?

This file is user facing https://docs.solo.io/gloo-edge/latest/reference/changelog/enterprise/

**Review by nfuden — APPROVED** [review]

**Review by jenshu — APPROVED** [review]

**Pull request merged**